packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -210,15 +210,6 @@ const builtInFlags = [
         description: 'Sets the build target e.g. node',
         link: 'https://webpack.js.org/configuration/target/#target',
     },
-    {
-        name: 'devtool',
-        usage: '--devtool <value>',
-        type: String,
-        negative: true,
-        alias: 'd',
-        description: 'Determine source maps to use',
-        link: 'https://webpack.js.org/configuration/devtool/#devtool',
-    },
```

Comment:
will `webpack-4` still support `webpack --devtool` ?

Reply:
seems like no then we should add it conditionally

Reply:
added not to drop it once webpack@4 is no longer supported